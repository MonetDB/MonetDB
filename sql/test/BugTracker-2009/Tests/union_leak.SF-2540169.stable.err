stderr of test 'union_leak.SF-2540169` in directory 'sql/test/BugTracker-2009` itself:


# 00:27:21 >  
# 00:27:21 >   mserver5 "--config=/ufs/manegold/_/scratch0/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert/etc/monetdb5.conf" --debug=10 --set gdk_nr_threads=0 --set "monet_mod_path=/ufs/manegold/_/scratch0/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert/lib64/MonetDB5:/ufs/manegold/_/scratch0/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert/lib64/MonetDB5/lib:/ufs/manegold/_/scratch0/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert/lib64/MonetDB5/bin" --set "gdk_dbfarm=/ufs/manegold/_/scratch0/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/manegold/_/scratch0/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert/var/MonetDB5/sql_logs"   --set mapi_open=true --set xrpc_open=true --set mapi_port=31321 --set xrpc_port=47339 --set xrpc_docroot=/net/rig.ins.cwi.nl/export/scratch0/manegold/Monet/Testing/Stable/source/sql --set monet_prompt= --trace  "--dbname=mTests_src_test_BugTracker-2009" --set mal_listing=0 "--dbinit= include sql;" ; echo ; echo Over..
# 00:27:21 >  

# builtin opt 	gdk_arch = 64bitx86_64-unknown-linux-gnu
# builtin opt 	gdk_version = 1.28.0
# builtin opt 	monet_pid = 28709
# builtin opt 	prefix = /ufs/manegold/_/scratch0/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert
# builtin opt 	exec_prefix = ${prefix}
# builtin opt 	gdk_dbname = tst
# builtin opt 	gdk_dbfarm = ${prefix}/var/MonetDB
# builtin opt 	gdk_debug = 8
# builtin opt 	gdk_mem_bigsize = 262144
# builtin opt 	gdk_alloc_map = yes
# builtin opt 	gdk_mem_pagebits = 14
# builtin opt 	gdk_vmtrim = yes
# builtin opt 	monet_admin = adm
# builtin opt 	monet_prompt = >
# builtin opt 	monet_welcome = yes
# builtin opt 	monet_mod_path = ${exec_prefix}/lib64/MonetDB:${exec_prefix}/lib64/bin
# builtin opt 	monet_daemon = yes
# builtin opt 	host = localhost
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_noheaders = no
# builtin opt 	mapi_debug = 0
# builtin opt 	mapi_clients = 2
# builtin opt 	sql_debug = 0
# builtin opt 	sql_logdir = ${prefix}/var/MonetDB
# builtin opt 	xquery_logdir = ${prefix}/var/MonetDB
# builtin opt 	standoff_ns = http://monetdb.cwi.nl/standoff
# builtin opt 	standoff_start = start
# builtin opt 	standoff_end = end
# config opt 	prefix = /ufs/manegold/_/scratch0/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert
# config opt 	config = ${prefix}/etc/monetdb5.conf
# config opt 	prefix = /ufs/manegold/_/scratch0/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert
# config opt 	exec_prefix = ${prefix}
# config opt 	gdk_dbfarm = ${prefix}/var/MonetDB5/dbfarm
# config opt 	gdk_dbname = demo
# config opt 	gdk_alloc_map = no
# config opt 	gdk_embedded = no
# config opt 	gdk_debug = 0
# config opt 	monet_mod_path = ${exec_prefix}/lib64/MonetDB5:${exec_prefix}/lib64/MonetDB5/lib:${exec_prefix}/lib64/MonetDB5/bin
# config opt 	monet_daemon = no
# config opt 	monet_welcome = yes
# config opt 	mero_msglog = ${prefix}/var/log/merovingian.log
# config opt 	mero_errlog = ${prefix}/var/log/merovingian.log
# config opt 	mero_timeinterval = 600
# config opt 	mero_pidfile = ${prefix}/var/run/merovingian.pid
# config opt 	mero_exittimeout = 7
# config opt 	mero_doproxy = yes
# config opt 	mero_discoveryttl = 600
# config opt 	mal_init = ${exec_prefix}/lib64/MonetDB5/mal_init.mal
# config opt 	mal_listing = 2
# config opt 	checkpoint_dir = ${prefix}/var/MonetDB5/chkpnt
# config opt 	mapi_port = 50000
# config opt 	mapi_open = false
# config opt 	sql_logdir = ${prefix}/var/MonetDB5/sql_logs
# config opt 	sql_init = ${exec_prefix}/lib64/MonetDB5/sql_init.sql
# cmdline opt 	config = /ufs/manegold/_/scratch0/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert/etc/monetdb5.conf
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	monet_mod_path = /ufs/manegold/_/scratch0/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert/lib64/MonetDB5:/ufs/manegold/_/scratch0/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert/lib64/MonetDB5/lib:/ufs/manegold/_/scratch0/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert/lib64/MonetDB5/bin
# cmdline opt 	gdk_dbfarm = /ufs/manegold/_/scratch0/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert/var/MonetDB5/dbfarm
# cmdline opt 	sql_logdir = /ufs/manegold/_/scratch0/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert/var/MonetDB5/sql_logs
# cmdline opt 	mapi_open = true
# cmdline opt 	xrpc_open = true
# cmdline opt 	mapi_port = 31321
# cmdline opt 	xrpc_port = 47339
# cmdline opt 	xrpc_docroot = /net/rig.ins.cwi.nl/export/scratch0/manegold/Monet/Testing/Stable/source/sql
# cmdline opt 	monet_prompt = 
# cmdline opt 	gdk_dbname = mTests_src_test_BugTracker-2009
# cmdline opt 	mal_listing = 0
# cmdline opt 	gdk_debug = 10
#warning: please don't forget to set your vault key!
#(see /ufs/manegold/_/scratch0/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert/etc/monetdb5.conf)

# 00:27:21 >  
# 00:27:21 >  ./union_leak.SF-2540169.SQL.sh union_leak.SF-2540169 
# 00:27:21 >  


# 00:27:21 >  
# 00:27:21 >  Mtimeout -timeout 60 mclient -lsql -umonetdb -Pmonetdb --host=rig --port=31321  /net/rig.ins.cwi.nl/export/scratch0/manegold/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert/mTests/src/test/BugTracker-2009/union_leak.SF-2540169.1.sql
# 00:27:21 >  


# 00:27:23 >  
# 00:27:23 >  Mtimeout -timeout 60 mclient -lmal -umonetdb -Pmonetdb --host=rig --port=31321 /net/rig.ins.cwi.nl/export/scratch0/manegold/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert/mTests/src/test/BugTracker-2009/union_leak.SF-2540169.2.mal
# 00:27:23 >  


# 00:27:23 >  
# 00:27:23 >  Mtimeout -timeout 60 mclient -lsql -umonetdb -Pmonetdb --host=rig --port=31321  /net/rig.ins.cwi.nl/export/scratch0/manegold/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert/mTests/src/test/BugTracker-2009/union_leak.SF-2540169.3.sql
# 00:27:23 >  


# 00:27:23 >  
# 00:27:23 >  Mtimeout -timeout 60 mclient -lmal -umonetdb -Pmonetdb --host=rig --port=31321 /net/rig.ins.cwi.nl/export/scratch0/manegold/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert/mTests/src/test/BugTracker-2009/union_leak.SF-2540169.2.mal
# 00:27:23 >  


# 00:27:23 >  
# 00:27:23 >  Mtimeout -timeout 60 mclient -lsql -umonetdb -Pmonetdb --host=rig --port=31321  /net/rig.ins.cwi.nl/export/scratch0/manegold/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert/mTests/src/test/BugTracker-2009/union_leak.SF-2540169.3.sql
# 00:27:23 >  


# 00:27:23 >  
# 00:27:23 >  Mtimeout -timeout 60 mclient -lmal -umonetdb -Pmonetdb --host=rig --port=31321 /net/rig.ins.cwi.nl/export/scratch0/manegold/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert/mTests/src/test/BugTracker-2009/union_leak.SF-2540169.2.mal
# 00:27:23 >  


# 00:27:23 >  
# 00:27:23 >  Mtimeout -timeout 60 mclient -lsql -umonetdb -Pmonetdb --host=rig --port=31321  /net/rig.ins.cwi.nl/export/scratch0/manegold/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert/mTests/src/test/BugTracker-2009/union_leak.SF-2540169.4.sql
# 00:27:23 >  


# 00:27:23 >  
# 00:27:23 >  Done.
# 00:27:23 >  

