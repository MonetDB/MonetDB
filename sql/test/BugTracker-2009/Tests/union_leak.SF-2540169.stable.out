stdout of test 'union_leak.SF-2540169` in directory 'sql/test/BugTracker-2009` itself:


# 00:27:21 >  
# 00:27:21 >   mserver5 "--config=/ufs/manegold/_/scratch0/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert/etc/monetdb5.conf" --debug=10 --set gdk_nr_threads=0 --set "monet_mod_path=/ufs/manegold/_/scratch0/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert/lib64/MonetDB5:/ufs/manegold/_/scratch0/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert/lib64/MonetDB5/lib:/ufs/manegold/_/scratch0/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert/lib64/MonetDB5/bin" --set "gdk_dbfarm=/ufs/manegold/_/scratch0/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/manegold/_/scratch0/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert/var/MonetDB5/sql_logs"   --set mapi_open=true --set xrpc_open=true --set mapi_port=31321 --set xrpc_port=47339 --set xrpc_docroot=/net/rig.ins.cwi.nl/export/scratch0/manegold/Monet/Testing/Stable/source/sql --set monet_prompt= --trace  "--dbname=mTests_src_test_BugTracker-2009" --set mal_listing=0 "--dbinit= include sql;" ; echo ; echo Over..
# 00:27:21 >  

# MonetDB server v5.10.0, based on kernel v1.28.0
# Serving database 'mTests_src_test_BugTracker-2009', using 4 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs dynamically linked
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008- MonetDB B.V., all rights reserved
# Visit http://monetdb.cwi.nl/ for further information
# Listening for connection requests on mapi:monetdb://rig.ins.cwi.nl:31321/
# MonetDB/SQL module v2.28.0 loaded

#function user.main():void;
#    clients.quit();
#end main;


# 00:27:21 >  
# 00:27:21 >  ./union_leak.SF-2540169.SQL.sh union_leak.SF-2540169 
# 00:27:21 >  


# 00:27:21 >  
# 00:27:21 >  Mtimeout -timeout 60 mclient -lsql -umonetdb -Pmonetdb --host=rig --port=31321  /net/rig.ins.cwi.nl/export/scratch0/manegold/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert/mTests/src/test/BugTracker-2009/union_leak.SF-2540169.1.sql
# 00:27:21 >  

[ 1000000	]
[ 1000000	]

# 00:27:23 >  
# 00:27:23 >  Mtimeout -timeout 60 mclient -lmal -umonetdb -Pmonetdb --host=rig --port=31321 /net/rig.ins.cwi.nl/export/scratch0/manegold/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert/mTests/src/test/BugTracker-2009/union_leak.SF-2540169.2.mal
# 00:27:23 >  

#-----------------#
# h	t	  # name
# str	lng	  # type
#-----------------#

# 00:27:23 >  
# 00:27:23 >  Mtimeout -timeout 60 mclient -lsql -umonetdb -Pmonetdb --host=rig --port=31321  /net/rig.ins.cwi.nl/export/scratch0/manegold/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert/mTests/src/test/BugTracker-2009/union_leak.SF-2540169.3.sql
# 00:27:23 >  

#SELECT sum(float1) AS currencyfloat FROM databasetest;
% .L1 # table_name
% currencyfloat # name
% double # type
% 24 # length
[ 99989206.4	]

# 00:27:23 >  
# 00:27:23 >  Mtimeout -timeout 60 mclient -lmal -umonetdb -Pmonetdb --host=rig --port=31321 /net/rig.ins.cwi.nl/export/scratch0/manegold/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert/mTests/src/test/BugTracker-2009/union_leak.SF-2540169.2.mal
# 00:27:23 >  

#-----------------#
# h	t	  # name
# str	lng	  # type
#-----------------#

# 00:27:23 >  
# 00:27:23 >  Mtimeout -timeout 60 mclient -lsql -umonetdb -Pmonetdb --host=rig --port=31321  /net/rig.ins.cwi.nl/export/scratch0/manegold/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert/mTests/src/test/BugTracker-2009/union_leak.SF-2540169.3.sql
# 00:27:23 >  

#SELECT sum(float1) AS currencyfloat FROM databasetest;
% .L1 # table_name
% currencyfloat # name
% double # type
% 24 # length
[ 99989206.4	]

# 00:27:23 >  
# 00:27:23 >  Mtimeout -timeout 60 mclient -lmal -umonetdb -Pmonetdb --host=rig --port=31321 /net/rig.ins.cwi.nl/export/scratch0/manegold/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert/mTests/src/test/BugTracker-2009/union_leak.SF-2540169.2.mal
# 00:27:23 >  

#-----------------#
# h	t	  # name
# str	lng	  # type
#-----------------#

# 00:27:23 >  
# 00:27:23 >  Mtimeout -timeout 60 mclient -lsql -umonetdb -Pmonetdb --host=rig --port=31321  /net/rig.ins.cwi.nl/export/scratch0/manegold/Monet/Testing/Stable/prefix.--enable-strict_--disable-optimize_--enable-debug_--enable-assert/mTests/src/test/BugTracker-2009/union_leak.SF-2540169.4.sql
# 00:27:23 >  


# 00:27:23 >  
# 00:27:23 >  Done.
# 00:27:23 >  

